.github/workflows/check-code-examples.yml

```diff
@@ -1,8 +1,4 @@
-
-# This workflow will install Python dependencies, run tests and lint with a single version of Python
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
-
-name: Python application
+name: Check code quality and examples on python3.8
```

Comment:
Typo? Since we later also check other python versions

Reply:
No it is intentional, we only run examples against python 3.8 to save test runtime.